packages/create-vite/src/index.ts

```diff
@@ -268,6 +268,19 @@ const FRAMEWORKS: Framework[] = [
       },
     ],
   },
+  {
+    name: 'marko',
+    display: 'Marko',
+    color: magenta,
+    variants: [
+      {
+        name: 'marko-run',
+        display: 'Marko Run ↗',
+        color: magenta,
+        customCommand: 'npm create marko@latest -- --name TARGET_DIR',
```

Comment:
It seems we have the same problem with https://github.com/vitejs/vite/pull/19573#discussion_r1980159075 here.
Merging main branch and updating this to 
```suggestion
        customCommand: 'npm create -- marko@latest --name TARGET_DIR',
```
should fix the problem.

Reply:
Oh! Good to know, thanks @sapphi-red. I added another commit that fixes this.